### PR TITLE
Chore/migrate to GitHub actions

### LINKED ISSUE #9: Migrate from Travis to GitHub actions
[Travis-ci](https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing) has not provided a free tier for open source projects for some time, so we need to migrate to GitHub actions, which is standard for our projects.

Migration guide: https://docs.github.com/en/actions/migrating-to-github-actions/migrating-from-travis-ci-to-github-actions

### PR DESCRIPTION
**Pull request by snim2**
Fixes #9

### CODE REVIEW

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+$config = \PhpCsFixer\Config::create()
+->setFinder(
+    PhpCsFixer\Finder::create()
+    ->exclude('vendor')
+    ->in(__DIR__)
+);
+// $fixer = new \PhpCsFixer\FixerFactory();
+// $fixer->registerBuiltInFixers()->useRuleSet(new \PhpCsFixer\RuleSet($config->getRules()));
```
Comment (RobjS) [inline .php-cs-fixer.php]: Are these commented-out lines included intentionally?

File: .php-cs-fixer.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (RobjS) [inline .php-cs-fixer.php]: Do we need both `.php-cs-fixer.php` and `.php_cs`? I can delete the latter and still run the linter successfully.

The docs for v2.19 only mention `php-cs-fixer.php`: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.19.3/doc/config.rst

File: .gitattributes [hunk 3]
```diff
@@ -1 +1 @@
-/composer.lock -diff
+/composer.lock text
```
Comment (RobjS) [inline .gitattributes:1]: 👍

**Review by RobjS — COMMENTED** [review]
One minor query around PHP CS Fixer config, otherwise all looks good 👍

File: .php-cs-fixer.php (hunk 1 above)
Comment (snim2) [reply .php-cs-fixer.php]: No, I've messed up a commit somewhere. I'll tidy that and force-push

**Comment by snim2** [comment]
@RobjS hopefully those issues are sorted now!

**Review by RobjS — APPROVED** [review]

**Pull request merged**